proof/crefine/RISCV64/Wellformed_C.thy

```diff
@@ -486,6 +486,22 @@ value_type irq_array_size = "Suc (unat Kernel_C.maxIRQ)"
 (* Input abbreviations for API object types *)
 (* disambiguates names *)
 
+text \<open>TCBFlags interface\<close>
+
+lemma scast_seL4_TCBFlag_simps[simp]:
+  "scast seL4_TCBFlag_NoFlag = 0"
+  "scast seL4_TCBFlag_fpuDisabled = tcbFlagToWord FpuDisabled"
+  by (clarsimp simp: seL4_TCBFlag_fpuDisabled_def seL4_TCBFlag_NoFlag_def tcbFlagToWord_def)+
+
+(* config_HAVE_FPU is unfolded because the C is defined with an ifdef, and so we just see the final
+   value instead of how it is constructed. *)
+lemma scast_seL4_TCBFlag_MASK_tcbFlagMask[simp]:
+  "scast seL4_TCBFlag_MASK = tcbFlagMask"
+  by (clarsimp simp: seL4_TCBFlag_MASK_def tcbFlagMask_def Kernel_Config.config_HAVE_FPU_def tcbFlagToWord_def)
+
+ (* end of Kernel_Config interface section *)
```

Comment:
I think previously seeing "Input abbreviations for API object types" indicated the end of the Kernel_Config section. The problem is that there isn't a *start* of the Kernel_Config interface section per se, it starts with "Domains", so not sure what I think of this comment now

Reply:
The `end of Kernel_Config interface section` comment came from `AARCH64`, which is where I made this change first. I'll do another pass and try to make the architectures a bit more consistent.

Reply:
Ah crap, fine to leave it in then. Happy for you to make it more consistent between arches though.

Reply:
Maybe not in this PR. I think having an official section would warrant a proper comment to set it up. As is stands, people can look for the end of it and add their stuff above